harness/determined/tensorboard/build.py

```diff
@@ -27,18 +27,18 @@ def get_rank_if_horovod_process_else_return_zero() -> Optional[int]:
     return int(os.getenv("HOROVOD_RANK", 0))
 
 
-def get_base_path(checkpoint_config: Dict[str, Any], manager: bool = False) -> pathlib.Path:
+def get_base_path(checkpoint_config: Dict[str, Any]) -> pathlib.Path:
+    allocation_id = os.environ.get("DET_ALLOCATION_ID", "")
     rank = get_rank_if_horovod_process_else_return_zero()
 
     if checkpoint_config.get("base_path"):
-        return pathlib.Path(checkpoint_config["base_path"]).joinpath("tensorboard")
-
-    if manager or rank == 0:
-        # In a distributed training job the manager should monitor the chief
-        # trials logs and ignore all other trials.
-        return pathlib.Path("/", "tmp", "tensorboard")
+        base_path = pathlib.Path(checkpoint_config["base_path"])
+    else:
+        base_path = pathlib.Path("/", "tmp")
```

Comment:
while we're here, we can just delete the `checkpoint_config.get("base_path")` case, it's _really_ old.

Reply:
A few tests apparently depend on the ability to override the tensorboard path.